tests/ext/closure_accessing_outside_variables.phpt

```diff
@@ -18,7 +18,8 @@ function setup($variable){
     });
 }
 
-(new Test())->m();
+// Cannot call a function while it is not traced and later expect it to trace
```

Comment:
I think we could keep the method if we had some way to prime the system. Marking this method as: 
"of interest" which means it will not be neg-cached. Or rather the cache will store some other marker.

Marking could be done by `-d` directive or env variable, or function we could call at startup.

This way we'll be able to not modify or delete existing tests - too much.
WDYT ?

Reply:
For this case, something like like `-d ddtrace.known_functions=Test::m`? We need something like this if we want to remove runtime tracing of internal functions anyway.

Reply:
We can cross this in another PR, I think.